Documentation/Blazorise.Docs/Pages/Docs/Extensions/Autocomplete/AutocompletePage.razor

```diff
@@ -173,7 +173,7 @@
         Placeholder for the empty search field.
     </DocsAttributesItem>
     <DocsAttributesItem Name="MinLength" Type="int" Default="1">
-        Minimum number of character needed to start search.
+        Minimum number of character needed to start search. Set this to 0 make the Autocomplete function like a dropdown.
```

Comment:
"to 0 **to** make..."
The documentation should also be reflected on the component itself, if you don't mind making that change also. Thanks!